apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LabelProviders/EnvironmentNameLogEntryLabelProvider.cs

```diff
@@ -38,7 +38,7 @@ public EnvironmentNameLogEntryLabelProvider(IHostingEnvironment hostingEnvironme
         /// <inheritdoc/>
         public void Invoke(Dictionary<string, string> labels)
         {
-            labels["aspnetcore_environment"] = _hostingEnvironment.EnvironmentName;
+            labels["aspnetcore_environment"] = _hostingEnvironment.EnvironmentName ?? "N/A";
```

Comment:
If the `EnvironmentName` is `null`, just don't add the label, similar to what you are doing on `UserLogEntryLabelProvider`.

Reply:
The downside of not adding labels (rather than using a default value) is that the logs without those labels don't show up when you include the label in a query. Logs without the `aspnetcore_environment` label won't show up when querying `labels.aspnetcore_environment!="Production"`. I think that in the case of this particular label (the hosting environment name) it's not that big of an issue as it's almost always said, but I've hit this "issue" a few times with my own labels. Anyway, I will change it.

Reply:
Thanks, I didn't know that!